src/lib/common/RenderFormat.h

```diff
@@ -46,10 +46,12 @@ typedef enum RenderFormat
   NO_FORMAT              = 0,
   NGSI_V1_LEGACY         = 1,
   NGSI_V2_NORMALIZED     = 2,
-  NGSI_V2_KEYVALUES      = 3,
-  NGSI_V2_VALUES         = 4,
-  NGSI_V2_UNIQUE_VALUES  = 5,
-  NGSI_V2_CUSTOM         = 6
+  NGSI_V2_SIMPLIFIEDNORMALIZED = 3,
+  NGSI_V2_SIMPLIFIEDKEYVALUES  = 4,
```

Comment:
Better not change existing numbers, adding the new ones at the end.

Reply:
Added in https://github.com/telefonicaid/fiware-orion/pull/4298/commits/1de26680286740a30ffeee899efe749412072f20